tests/integration/external/guzzle6/test_cat.php

```diff
@@ -64,6 +64,7 @@
     [{"name":"Supportability/library/Guzzle 4-5/detected"},
                                                           [1,    0,    0,    0,    0,    0]],
     [{"name":"Supportability/library/Guzzle 6/detected"}, [1,    0,    0,    0,    0,    0]],
+    [{"name":"Supportability/library/Guzzle 7/detected"}, [1,    0,    0,    0,    0,    0]],
```

Comment:
For the Guzzle 6 cases, is there some limitation such that we need to have Guzzle 7 detected?

Reply:
In [unpack_guzzle.php](https://github.com/newrelic/newrelic-php-agent/blob/9fd43b56a296e761156db9aa08a1919a975cd56c/tests/include/unpack_guzzle.php#L27), this file extracts the different guzzle packages in the tar file (4-7) and imports the correct guzzle version. Then in [php_execute.c](https://github.com/newrelic/newrelic-php-agent/pull/498/files#diff-beedad7bbfc9687348e557b4752fbf6f54e4407a6e3177e8cabaaec7e3ffd625), we define the nr_guzzle6_enable and nr_guzzle7_enable functions, which is the reason why guzzle 6 and 7 are both detected in the Guzzle 6 test cases. I will further look into this and see if I can prevent guzzle 7 from being detected